### PR TITLE
fixes apollo debug & warn log, control log out by slf4j xml file.

### PR DESCRIPTION
**Pull request by dongzl**
fixes apollo debug & warn log, control log out by slf4j xml file.

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml [hunk 1]
```diff
@@ -48,5 +48,11 @@
             <version>${apollo.client.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+            <version>${slf4j.version}</version>
+            <scope>provided</scope>
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: Should jul-to-slf4j be compile?

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml [hunk 2]
```diff
@@ -48,5 +48,11 @@
             <version>${apollo.client.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+            <version>${slf4j.version}</version>
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: Should move it to root pom's `dependencyManagement`

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml (hunk 1 above)
Comment (dongzl) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: Thanks, I have fixed it.

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml (hunk 2 above)
Comment (dongzl) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: Thanks, I have fixed it.

File: sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml [hunk 3]
```diff
@@ -48,5 +48,10 @@
             <version>${apollo.client.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jul-to-slf4j</artifactId>
+            <scope>compile</scope>
```
Comment (terrymanu) [inline sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: `compile` is a default scope, we do not need add it if scope is `compile`

Comment (dongzl) [reply sharding-orchestration/sharding-orchestration-config/sharding-orchestration-config-apollo/pom.xml]: Thanks, I have fixed it and remember.

**Pull request merged**